ibis/pyspark/compiler.py

```diff
@@ -1040,12 +1013,12 @@ def compile_lead(t, expr, scope, *, window, **kwargs):
 
 @compiles(ops.MinRank)
 def compile_rank(t, expr, scope, *, window, **kwargs):
-    return F.rank().over(window).astype('long') - F.lit(1)
```

Comment:
for my own edification, what does F.lit(1) do in pyspark?

Reply:
F.lit(1) creates a literal column of "1"s